tests/data/base/Speaker.php

```diff
@@ -40,6 +40,7 @@ public function scenarios()
     {
         return [
             'test' => ['firstName', 'lastName', '!underscore_style'],
+            'duplicates' => ['firstName', 'firstName', '!underscore_style', '!underscore_style'],
```

Comment:
I wasn't really sure if by adding the duplicates to the test scenario I would break any existing tests, so I just added a new scenario. Do you think it's better just to add the attributes to the test scenario?